lib/content/audio.ex

```diff
@@ -19,6 +19,6 @@ defprotocol Content.Audio do
   @doc "Converts an audio struct to the mid/vars params for the PA system"
   @spec to_params(Content.Audio.t()) :: value()
   def to_params(audio)
-  @spec to_tts(Content.Audio.t()) :: String.t()
+  @spec to_tts(Content.Audio.t()) :: {String.t(), [{String.t(), String.t(), integer()}] | nil}
```

Comment:
question: Is the idea to phase this function out? Looks like all calls to it were replaced with `tts_text` calls, so mostly just making sure that was intentional.

Reply:
I should have mentioned in the description: This function isn't currently used, but it will be in a subsequent PR that implements the code path for sending to Scully.